app/assets/stylesheets/views/profile.scss

```diff
@@ -167,3 +167,7 @@
     border-bottom: unset;
   }
 }
+
+.align-self-flex-end {
```

Comment:
Refactor (blocking): This is no longer required as you're using the utility CSS classes as per https://github.com/forem/forem/pull/15255#discussion_r740844226